src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -604,5 +628,9 @@ public void CloseWindow()
         {
             RequestCloseWindow();
         }
+        public void CloseWindowPreserve(bool isCheckboxChecked)
```

Comment:
The overload does not seem to work in Javascript context, I am getting this error: `Invalid number of parameters` in console. It is probably getting confused or maybe overloads don’t work in script object. The other solution is to use a new method. I have update the PR please take a look. @QilongTang